### ISSUE TITLE
Release 2021-01-03 Typos

### ISSUE THREAD

**Issue by linker81** [probml/pml-book#32]
Print Page 589, PDF Page 619: "A natural approach to ~~transfer learning~~ meta-learning"

Print Page 590, PDF Page 620: "N-way K-shot classiﬁcation , in which the system is expected to learn to classify ~~K~~N classes using just ~~N~~K training examples of each class.".  
Based on the figure 19.7 and the next example I think it should be rewritten as suggested.

**Comment by StevenBBryant** [comment]
PDF Page 3 – “In image classification, the input space X is the set of images, which is a very high dimensional space: for a color image of size D1 × D2 pixels, we have X = R D, where D = 3 × D1 × D2. [Comment: It is not clear where “3” comes from. From the preceding text, on page 3, one might interpret 3 to represent “setosa, versicolor and virginica” rather than Red, Blue, Green.]

PDF Page 3 – “When the inputs are structured objects (e.g., sequences of words, or variable-shaped graphs), rather than fixed-length vectors, the data is usually stored in some other format rather than in a design matrix” [Comment: Should the word “structured” be “unstructured”? Perhaps what is needed is a definition of un/structured objects vs. un/structured data.] 

PDF Page 4 – Figure 1.4 Caption – “This can be normalized to get a distribution over class labels for each node” [Comment: It is not clear to what “This” refers.]

PDF Page 5 – “This is a very simple example of a classifier, in which we have partitioned the input space into two regions, defined by the 1d decision boundary…”  [Comment: Should the text say “one dimensional (1d)” to clarify the meaning of “1d”?. This will also give an implied meaning to “2d” which appears later in the text.]

PDF Page 6 – [Comment: The placement of Table 1.2, such that it immediately follows Equation 1.2 is confusing. I think Table 1.2 would be better placed following the text that read “In this case, we might use the asymmetric loss function … shown in Table 1.2.” (prior to Equation 1.4).]

PDF Page 6 – “One way to define the problem of model fitting or training is to find a setting of the paramters that minimizes the empirical risk on the training set:” [Comment: “paramters” should be “parameters”.]

PDF Page 7 – “For example, in Fig. 1.4b„ we see…” [Comment: There are two commas following “1.4b”]

PDF Page 7 – “We can capture our uncertainty using a conditional probability distribution” [Comment: Should there be some punctuation (eg, “:”) at the end of this statement before introducing Equation 1.7? This occurs multiple times in the chapter, so it may be there is correct and/or intentional.]

**Comment by ducminhkhoi** [comment]
+ Chapter 2, page 22, the paragraph before Eq. (2.3), we we expect to see if H = h; (duplicate we)
+ Chapter 2, page 29, paragraph 2, “numbers ending in 9”,. (remove ,)
+ Chapter 2, Fig. 2.4, the description should be: posterior, likelihood, prior (top-down direction). Also, the x-axis is reversed (from left to right is 0 to 35 instead of 35 to 0)
+ Chapter 2, page 32, paragraph 1, line 3, duplicate we
+ Chapter 2, page 34, the last sentence: generalizatino -> generalization
+ Chapter 2, page 35, the first sentence: horizonta -> horizontal

**Comment by murphyk** [comment]
fixed
